imports/plugins/core/ui-navbar/client/containers/navbar.js

```diff
@@ -36,4 +37,23 @@ function composer(props, onData) {
 
 registerComponent("NavBar", NavBar, composeWithTracker(composer));
 
+registerComponent("NavBarCheckout", (props, context) => {
```

Comment:
One component per file. Also, separate the registration of the component and the definition of the component like we do in other places.

Reply:
Done.